frontstage/views/surveys/help/technical_help.py

```diff
@@ -85,5 +87,27 @@ def _send_new_message(subject, party_id):
 
     response = conversation_controller.send_message(json.dumps(message_json))
 
-    logger.info("Secure message sent successfully", message_id=response["msg_id"], party_id=party_id)
+    category = form["category"].data if "category" in form else None
+    collection_exercise_id = (
+        flask_session["collection_exercise_id"] if "collection_exercise_id" in flask_session else None
+    )
+
+    period_ref = flask_session["period_ref"] if "period_ref" in flask_session else None
+
+    survey_id = form["survey_id"].data if "survey_id" in form else None
+
+    survey_ref = flask_session["survey_ref"] if "survey_ref" in flask_session else None
```

Comment:
Will need to test if this results in the expected behaviour. We might want to default this to `0000` or something so it doesn't mis-report against a particular survey.